### ISSUE TITLE
MNU doing changes as user without code modification ... as non-SystemUser (DataCurator)

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#99]
It looks like `RowanProjectService >> changes` is doing `GsCurrentSession currentSession resolveSymbol: #JadeServer) value.`, instead of doing an indirect lookup of the JadeServer class.

Here's the stack:
```
a MessageNotUnderstood occurred (error 2010), a UndefinedObject does not understand  #'value'
---
MessageNotUnderstood >> defaultAction @2 line 3
MessageNotUnderstood (AbstractException) >> _signalWith: @5 line 25
MessageNotUnderstood (AbstractException) >> signal @2 line 47
UndefinedObject (Object) >> doesNotUnderstand: @9 line 10
UndefinedObject (Object) >> _doesNotUnderstand:args:envId:reason: @7 line 12
RowanProjectService >> changes @5 line 5
[] in  ExecBlock1 (JadeServer) >> updateFromSton: @9 line 11
Array (Collection) >> do: @5 line 10
JadeServer64bit32 (JadeServer) >> updateFromSton: @7 line 6
GsNMethod class >> _gsReturnToC @1 line 1

**Comment by ericwinger** [comment]
I believe that the proper lookup is 
```
Rowan platform jadeServerClassNamed: #JadeServer.

**Comment by ericwinger** [comment]
Fixed in oscar branch ea0af45
